webapp/data_structures/FROMWSB.js

```diff
@@ -53,7 +53,7 @@ function FROMWSB() {
                 state: new ScilabDouble(),
                 dstate: new ScilabDouble(),
                 odstate: list(),
-                rpar: new ScilabDouble(),
+                rpar: new ScilabDouble([]),
                 ipar: new ScilabDouble([1], [-31], [1], [1], [0]),
```

Comment:
```suggestion
                ipar: new ScilabDouble([this.varnam.length],..._str2code(this.varnam),[this.Method],[this.ZC],[this.OutEnd]),
```

Suggestion for consistency